grammars/ini.cson

```diff
@@ -1,5 +1,6 @@
 'fileTypes': [
   'cfg',
+  'conf',
```

Comment:
No worries. The only thing I _think_ we can do here is rely on the inbuilt grammar selector (http://flight-manual.atom.io/using-atom/sections/grammar/). This should allow users to select the INI grammar for `.conf` files.